connectivity/connectivity-demos-test/src_test/com/axonivy/connectivity/rest/TestPersonClient.java

```diff
@@ -0,0 +1,50 @@
+package com.axonivy.connectivity.rest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import javax.ws.rs.core.Response;
+
+import org.junit.jupiter.api.Test;
+
+import com.axonivy.connectivity.Person;
+import com.axonivy.connectivity.rest.client.PersonClient;
+
+import ch.ivyteam.ivy.environment.IvyTest;
+
+@IvyTest
```

Comment:
Should we add a small javadoc to explain the usage of IvyTest here? 
Since this is not an official example project it may not be necessary. But still it is on the public GIT HUB repo.
It's up to you if you want to add something here or not.

Reply:
👍